ckanext/unaids/theme/templates/package/read.html

```diff
@@ -3,6 +3,25 @@
 {% set pkg = c.pkg_dict %}
 
 {% block package_description %}
+
+  {% if pkg.org_to_allow_transfer_to %}
+    <div class="alert alert-info" style="margin-top: -10px;">
+      {% if h.check_organization_update_access(pkg.org_to_allow_transfer_to) %}
```

Comment:
Your new auth function can be called properly with `h.check_access('organisation_editor`, {'org_id': pkg.org_to_allow_transfer_to})` I think.